recipe/meta.yaml

```
@@ -20,7 +20,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 5
+  number: 6
   binary_relocation: true
```

```suggestion
  binary_relocation: true
  skip: True  # [py!=3.12]
```
This should have the effect of skipping versions of python except 3.12.

---

I'd expect this suggestion to build only py312 for the whole recipe, right?

What's needed is to (1) build output `gdma` _once_ with any python present at buildtime (to process the version file) and incurring no runtime python dependence and then (2) build output `pygdma` multiple times (once for each python version). The current way was the simplest way to avoid invoking all the extra logic that happens in c-b/c-f when python's involved. If it's not too much of an anti-pattern, I don't mind bumping the hard-coded version every couple years; that is, leaving it as-is.

---

Since this is only a build-time dependency, I think I have a different solution to avoid an unneeded build matrix here.   I have to test it locally but I'll follow up shortly!